package.json

```diff
@@ -20,7 +20,7 @@
 	},
 	"repository": {
 		"type": "git",
-		"url": "git://github.com/minimistjs/minimist.git"
+		"url": "https://github.com/minimistjs/minimist.git"
```

Comment:
git+ssh instead please

Reply:
Changed as requested - though as per https://github.com/inspect-js/has-symbols/pull/22#discussion_r2274531112 I think `https://` would be better.